ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
+    "CaseFieldID": "specDefenceRouteAdmittedAmountClaimedLabel",
+    "PageDisplayOrder": 5,
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "FieldShowCondition": "respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\"",
+    "PageID": "defenceAdmittedRoute",
```

Comment:
Can we please rename it to defencePartAdmittedRoute to distinguish it from the future 'Admit all claim' journey

Reply:
Changed from defenceAdmittedRoute to defenceAdmittedPartRoute for this journey